sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 ### Features Added
   - Added support for IP-based access control to default and hub workspaces. 
   - Add support for additional include in spark component.
+  - Removing preview tag for following connections type
+    - Azure AI Search
+    - Azure AI Content Safety
+    - Azure AI Speech
+    - Azure Blob Storage
+    - Azure Datalake Storage Gen 2
```

Comment:
I can't find this in the connection subtypes

Reply:
![image](https://github.com/user-attachments/assets/ad9e125c-31b3-4239-8a0d-db0cca25d204)